src/instructlab/model/chat.py

```diff
@@ -55,6 +67,211 @@
 PROMPT_PREFIX = ">>> "
 
 
+@click.command()
+@click.argument(
+    "question",
+    nargs=-1,
+    type=click.UNPROCESSED,
+)
+@click.option(
+    "-m",
+    "--model",
+    default=helpers.DEFAULT_MODEL,
+    show_default=True,
+    help="Model to use",
+)
+@click.option(
+    "-c",
+    "--context",
+    default="default",
+    show_default=True,
+    help="Name of system context in config file.",
+)
+@click.option(
+    "-s",
+    "--session",
+    type=str,
+    default=None,
+    help="Filepath of a dialog session file.",
+)
+@click.option(
+    "-qq",
+    "--quick-question",
+    is_flag=True,
+    help="Exit after answering question.",
+)
+@click.option(
+    "-gm",
+    "--greedy-mode",
+    is_flag=True,
+    help="Use model greedy decoding. Useful for debugging and reproducing errors.",
+)
+@click.option(
+    "--max-tokens",
+    type=click.INT,
+    help="Set a maximum number of tokens to request from the model endpoint.",
+)
+@click.option(
+    "--endpoint-url",
+    type=click.STRING,
+    help="Custom URL endpoint for OpenAI-compatible API. Defaults to the `ilab serve` endpoint.",
+)
+@click.option(
+    "--api-key",
+    type=click.STRING,
+    default=DEFAULT_API_KEY,  # Note: do not expose default API key
+    help="API key for API endpoint. [default: config.DEFAULT_API_KEY]",
+)
+@click.option(
+    "--tls-insecure",
+    is_flag=True,
+    help="Disable TLS verification.",
+)
+@click.option(
+    "--tls-client-cert",
+    type=click.Path(),
+    default="",
+    show_default=True,
+    help="Path to the TLS client certificate to use.",
+)
+@click.option(
+    "--tls-client-key",
+    type=click.Path(),
+    default="",
+    show_default=True,
+    help="Path to the TLS client key to use.",
+)
+@click.option(
+    "--tls-client-passwd",
+    type=click.STRING,
+    default="",
+    help="TLS client certificate password.",
+)
+@click.option(
+    "--model-family",
+    default="merlinite",
+    show_default=True,
+    help="model family to use when picking a chat template",
+)
+@click.pass_context
+def chat(
+    ctx,
+    question,
+    model,
+    context,
+    session,
+    quick_question,
+    greedy_mode,
+    max_tokens,
+    endpoint_url,
+    api_key,
+    tls_insecure,
+    tls_client_cert,
+    tls_client_key,
+    tls_client_passwd,
+    model_family,
+):
+    """Run a chat using the modified model"""
```

Comment:
Not necessarily a modified model no?

Reply:
This was the help text from before, not looking to change stuff like that in my PR I believe